app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -10,8 +10,10 @@ layout: learn/tutorial.html
 subheader_select: tutorials
 main_css_id: learn
 description: "Learn to create advanced JavaFX animations"
-last_update: 2024-05-22
+last_update: 2024-05-31
+author: ["ConnorSchweighöfer"]
 ---
+<br>
```

Comment:
Is there a reason for this line break here?

Reply:
Yea, this is intentional. It looks weird without it:

Without:
![without](https://github.com/java/devjava-content/assets/88111627/358e8d24-ac98-4973-b899-c1001be25a85)

With:
![with](https://github.com/java/devjava-content/assets/88111627/2ecb49e8-34a9-47df-aed0-9c7dc77c62fe)

Reply:
Yeah I just realized that as well (see the edit which I wrote pretty much at the same time as your comment for alternatives)

Reply:
> Yeah I just realized that as well (see the edit which I wrote pretty much at the same time as your response for alternatives)

I see, but I was comparing it to other sites like this `https://dev.java/learn/javafx/layout/` and it looks different if I have 
`<a id="intro">&nbsp;</a>` or not above the `## Introduction` and because I dont create the table of contents manually I added a `<br>` instead:

Without <a>:
![image](https://github.com/java/devjava-content/assets/88111627/9b70783c-ee98-4c50-a527-798446d95b00)

With <a>:
![image](https://github.com/java/devjava-content/assets/88111627/0a0a7287-2d2e-41a0-9130-e8c03845fecd)